mobile/src/main/java/ch/epfl/sdp/mobile/ui/prepare_game/PrepareGameScreenState.kt

```diff
@@ -0,0 +1,19 @@
+package ch.epfl.sdp.mobile.ui.prepare_game
+
+interface PrepareGameScreenState {
+  var colorChoice: ColorChoice
+  var gameType: GameType
+
+  val onNewLocalGame: () -> Unit
+  val onNewOnlineGame: () -> Unit
+}
+
+enum class ColorChoice {
+  WHITE,
+  BLACK
+}
+
+enum class GameType {
+  LOCAL,
+  ONLINE
+}
```

Comment:
I think we agreed on using `PascalCase` for enums and constants, since this is the convention in Jetpack Compose.

Reply:
Ok